functions.global.php

```diff
@@ -484,3 +484,36 @@ function jetpack_is_mobile( $kind = 'any', $return_matched_agent = false ) {
 	 */
 	return apply_filters( 'jetpack_is_mobile', $return, $kind, $return_matched_agent );
 }
+
+/**
+ * Determine whether the current request is for accessing the frontend.
+ *
+ * @return bool True if it's a frontend request, false otherwise.
+ */
+function jetpack_is_frontend() {
+	$is_frontend = true;
+
+	if (
+		is_admin() ||
+		wp_doing_ajax() ||
+		wp_doing_cron() ||
+		wp_is_json_request() ||
+		wp_is_jsonp_request() ||
+		wp_is_xml_request() ||
+		is_feed() ||
+		( defined( 'REST_REQUEST' ) && REST_REQUEST ) ||
```

Comment:
I'm seeing an error log warning on dotcom, `Warning: Use of undefined constant REST_REQUEST - assumed 'REST_REQUEST' (this will throw an Error in a future version of PHP) in /home/wpcom/public_html/wp-content/mu-plugins/jetpack/functions.global.php on line 446`.

Reply:
Oh, good eye! I had to manually sync the changes to the WP.com diff (Fusion was having a hard time trying it) and definitely copied the wrong code there :)

Here we have `( defined( 'REST_API_REQUEST' ) && REST_API_REQUEST ) ||` but in the diff I used the wrong constant: `( defined( 'REST_API_REQUEST' ) && REST_REQUEST ) ||`. 🤦 

I already updated the diff with the correct variable 👍